rmgpy/tools/canteraModel.py

```diff
@@ -316,7 +316,8 @@ def plot(self, data):
             # plot
             GenericPlot(xVar=time, yVar=TData).plot('{0}_temperature.png'.format(i+1))
             GenericPlot(xVar=time, yVar=PData).plot('{0}_pressure.png'.format(i+1))
-            SimulationPlot(xVar=time, yVar=speciesData, ylabel='Mole Fraction').plot(os.path.join(self.outputDirectory,'{0}_mole_fractions.png'.format(i+1)))
+            SimulationPlot(xVar=time, yVar=speciesData[:topSpecies], ylabel='Mole Fraction').plot(os.path.join(self.outputDirectory,'{0}_mole_fractions.png'.format(i+1)))
+            
```

Comment:
so actually the `topSpecies` is kind of confusing because it won't plot the top concentrated species, at least the species not sorted by default. Here's the plot it produced for my own example mechanism.
![1_mole_fractions_new](https://cloud.githubusercontent.com/assets/2739496/16159884/37e672a6-3493-11e6-871b-917f042ee482.png)

The species being plotted are almost zero-concentration, all the products that are in high concentration are not in the plot.